examples/mlperf/helpers.py

```diff
@@ -4,6 +4,9 @@
 from tinygrad.nn import state
 from tinygrad.tensor import Tensor
 from tinygrad.dtype import dtypes
+from tinygrad.nn.state import get_state_dict
+
+from extra.models.bert import BertForMLPerf
```

Comment:
generally the direction should be helpers imports nothing and individual example imports from target. if it's only used in bert, it stays in bert